src/App.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+import serverTodos from './api/todos';
+import serverUsers from './api/users';
+
+const preparedTodos: Todo[] = serverTodos.map(todo => ({
+  ...todo,
+  user: serverUsers.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => (
   <div className="App">
-    <h1>Static list of todos</h1>
+    <TodoList preparedTodos={preparedTodos} />
```

Comment:
best practice
TodoList shouldn't know you prepare data or get it already prepared
```suggestion
    <TodoList todos={preparedTodos} />
```